app/questionnaire/placeholder_parser.py

```diff
@@ -0,0 +1,65 @@
+from app.data_model.answer_store import AnswerStore
+from app.data_model.answer import Answer
+from app.questionnaire.placeholder_transforms import PlaceholderTransforms
+
+
+class PlaceholderParser:
+    """
+    Parses placeholder statements from a schema dict and returns a map of their
+    final values
+    """
+    def __init__(self, answer_store=None, metadata=None, lists=None):
+        self._lists = lists
+        self._answer_store = AnswerStore({}) if answer_store is None else answer_store
```

Comment:
Do we need to pass anything into `AnswerStore` can it not just be `AnswerStore()` as it gets set to a `defaultdict(list)` in the `__init__` if none is provided which differs from what is being set here.

Reply:
OK